core/tests/test_utils.py

```diff
@@ -263,6 +362,22 @@ def _send_post_request(
     def post_email(
             self, recipient_email, sender_email, subject, body, html_body=None,
             expect_errors=False, expected_status_int=200):
+        """EmailMessage Class is used for creating the email message.
+            The email backend is then responsible for sending the email.
+
+        Args:
+            recipient_email: A list or tuple of recipient addresses.
+            sender_email: The sender's address.
```

Comment:
Please specify the type of the arguments, i think this and html_body it should be string?